Pipeline: input: elasticsearch: style (#1720)
fluent/fluent-bit-docs: pipeline/inputs/elasticsearch.md

Ingesting from beats series agents is also supported.
For example, [Filebeats](https://www.elastic.co/beats/filebeat), [Metricbeat](https://www.elastic.co/beats/metricbeat), and [Winlogbeat](https://www.elastic.co/beats/winlogbeat) are able to ingest their collected data through this plugin.

The Fluent Bit node information is returning as Elasticsearch 8.0.0.

> Users have to specify the following configurations on their beats configurations:

Author: ```suggestion
Users must specify the following configurations on their beats configurations:
```
